deploy/examples/cluster-on-pvc.yaml

```diff
@@ -132,7 +132,7 @@ spec:
                 requests:
                   storage: 10Gi
               # IMPORTANT: Change the storage class depending on your environment
-              storageClassName: gp2
+              storageClassName: gp2-csi
               volumeMode: Block
               accessModes:
                 - ReadWriteOnce
```

Comment:
Update the cluster-stretched-aws.yaml

plus there is a pvc-cluster.md docuemtnation file,
If you do a search of `gp2` you will find couple of instances, so update wherever they are related

Reply:
I have updated the pvc-cluster.md documentation file in places which were related with the previous changes. Please do let me know if any other changes need to be made in this file.

Reply:
You can also change
create_test.go -> integration tests && root-test-framework.md
ceph-cluster-crd.md ->doc